mmdet3d/core/visualizer/open3d_vis.py

```diff
@@ -57,6 +57,7 @@ def _draw_points(points,
     return pcd, points_colors
 
 
+# NOTE(swc): draw bboxes on the pcd
```

Comment:
remove note unless the note is useful, you can use `git add -p` to select changes in a file.